src/RuleImportAsset.php

```diff
@@ -1082,7 +1088,7 @@ public static function getItemTypesForRules()
 
         $types = [];
         foreach ($CFG_GLPI["ruleimportasset_types"] as $itemtype) {
-            if (class_exists($itemtype)) {
+            if (class_exists($itemtype) && is_a($itemtype, CommonDBTM::class, true)) {
```

Comment:
The first check is included in the second.
```suggestion
            if (is_a($itemtype, CommonDBTM::class, true)) {
```